.github/workflows/ci.yml

```diff
@@ -63,6 +64,9 @@ jobs:
         if: contains(matrix.platform.target, 'android')
         run: cargo install cargo-apk
 
+      - name: Download dependencies
+        run: cargo $CMD fetch
+
```

Comment:
dependencies will be downloaded when building anyways? is there a reason to separate this into its own step?

Reply:
removed on commit 71bb9a2